wasm.md

```diff
@@ -651,11 +649,13 @@ When globals are declared, they must also be given a constant initialization val
     rule asGMut (      T:ValType   ) => const T
 
     syntax Defn       ::= GlobalDefn
-    syntax GlobalDefn ::= "(" "global"               TextGlobalType Instr ")"
-                        | "(" "global" TextFormatIdx TextGlobalType Instr ")"
+    syntax GlobalDefn ::= "(" "global"            TextGlobalType Instr ")"
+                        | "(" "global" Identifier TextGlobalType Instr ")"
                         |     "global" GlobalType
  // ---------------------------------------------
-    rule <k> ( global ID:TextFormatIdx TYP:TextGlobalType IS:Instr ) => ( global TYP IS ) ... </k>
+    rule asGMut ( (mut T:ValType ) ) => var   T
+    rule asGMut (      T:ValType   ) => const T
```

Comment:
asGMut is already defined above, why are you redefining it here?

Reply:
Well it should be some merging issue..